src/WsSecurity.php

```diff
@@ -8,35 +8,50 @@ class WsSecurity
      * @var Security
      */
     protected $security;
+
     /**
      * @param string $username
      * @param string $password
-     * @param bool $passwordDigest
-     * @param int $addCreated
-     * @param int $addExpires
-     * @param bool $mustunderstand
+     * @param bool   $passwordDigest
+     * @param int    $addCreated
+     * @param int    $addExpires
+     * @param bool   $mustunderstand
      * @param string $actor
      * @param string $usernameId
-     * @param bool $addNonce
+     * @param bool   $addNonce
+     * @param string $envelopeNamespace
      */
-    protected function __construct($username, $password, $passwordDigest = false, $addCreated = 0, $addExpires = 0, $mustunderstand = false, $actor = null, $usernameId = null, $addNonce = true)
-    {
+    protected function __construct(
+        $username,
+        $password,
+        $passwordDigest = false,
+        $addCreated = 0,
+        $addExpires = 0,
+        $mustunderstand = false,
+        $actor = null,
+        $usernameId = null,
+        $addNonce = true,
+        $envelopeNamespace = 'SOAP-ENV'
+    ) {
         $this
-            ->initSecurity($mustunderstand, $actor)
+            ->initSecurity($mustunderstand, $actor, $envelopeNamespace)
             ->setUsernameToken($username, $usernameId)
             ->setPassword($password, $passwordDigest, $addCreated)
             ->setNonce($addNonce)
             ->setCreated($addCreated)
             ->setTimestamp($addCreated, $addExpires);
     }
+
     /**
-     * @param bool $mustunderstand
+     * @param bool   $mustunderstand
      * @param string $actor
+     * @param string $envelopeNamespace
+     *
      * @return WsSecurity
      */
-    protected function initSecurity($mustunderstand = false, $actor = null)
+    protected function initSecurity($mustunderstand = false, $actor = null, $envelopeNamespace)
     {
-        $this->security = new Security($mustunderstand, $actor);
+        $this->security = new Security($mustunderstand, $actor, $envelopeNamespace);
```

Comment:
According to my previous suggestion
```suggestion
        $this->security = new Security($mustunderstand, $actor, Security::NS_WSSE, $envelopeNamespace);
```